app/templates/components/dashboard/unread-groups.hbs

```diff
@@ -35,7 +35,7 @@
     {{/if}}
   </ul>
 {{else if (not session.hasUser)}}
-  Anyone can create and discover groups. Each group centers around a topic or activity.
+  {{t "groups.index.call-to-action"}}
   <a href="#" {{action "signUp"}}>Join Kitsu</a> to get started!
```

Comment:
It doesn't include the full text.

Reply:
I didn't include the link because:
1) The first string was already translated in another file (logged in user)
2) Same as before, couldn't get the "action" part working

Reply:
Yeah, the action rather than a direct link makes it a little difficult.

Ignore this, we'll probably have to create a dedicated link to open the sign up modal to fix it issues like this.

Reply:
Okey, ignoring for now